src/hooks/api/friends/useCreateFriendsMutation.ts

```
@@ -0,0 +1,17 @@
+export const useCreateFriendsMutation = () => {
+  const queryClient = useQueryClient();
+
+  return useMutation({
+    mutationFn: (data: Friends) => postFriends(data),
+    onSuccess: () =>
+      queryClient.invalidateQueries({
+        queryKey: ['friends'],
+      }),
+  });
+
+  return { ...mutation };
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)